app/src/main/res/layout/faq_content.xml

```diff
@@ -35,7 +35,7 @@
         android:paddingTop="16dp"
         android:paddingBottom="16dp"
         android:text="@{viewModel.question}"
-        android:textColor="@color/oppia_primary_text"
+        android:textColor="@color/component_color_faq_activity_text_color"
         android:textSize="14sp" />
 
       <View
```

Comment:
I think you should change the **oppia_faq_divider** to **color_def_dark_silver** as we are shifting colors from **colors_migrating.xml** to **color_defs.xml**, Thanks.

Reply:
> I think you should change the **oppia_faq_divider** to **color_def_dark_silver** as we are shifting colors from **colors_migrating.xml** to **color_defs.xml**, Thanks.

Develop uses oppia_faq_divider, although I tried changing the color and this is how it turned out to be--
| oppia_faq_divider  | color_def_dark_silver  |
|-------|---|
|<img src="https://user-images.githubusercontent.com/78796264/168992695-14891c49-8284-40b1-9141-b17093830b52.jpeg" width="290" />|<img src="https://user-images.githubusercontent.com/78796264/168992683-b55e702c-9d4d-4b34-8ffd-509dd4c8ab5b.jpeg" width="290" />|

Reply:
No Bhakti **oppia_faq_divider** and **color_def_dark_silver** have the same hex codes so I think there should be no changes in UI.

Reply:
Here it was my fault in naming the color I apologize for that and now I have fixed that.

Reply:
No worries. Thanks.